matminer/featurizers/site.py

```diff
@@ -622,3 +623,55 @@ def citations(self):
 
     def implementors(self):
         return ['Qi Wang']
+
+
+class EwaldSiteEnergy:
+    """Compute site energy from Coulombic interactions
+
+    User notes:
+        - This class uses that `charges that are already-defined for the structure`.
+          Consider assigning them using the ``add_charges_*`` from ``pymatgen``
+          before calling this class
+
+        - Ewald summations can be expensive. If you evaluating every site in many
+          large structures, run all of the sites for each structure at the same time.
+          We cache the Ewald result for the structure that was run last, so looping
+          over sites and then structures is faster than structures than sites.
+
+    Features:
+        ewald_site_energy - Energy for the site computed from Coulombic interactions"""
+
+    def __init__(self, accuracy=None):
+        """
+        Args:
+            accuracy (int): Accuracy of Ewald summation, number of decimal places
+        """
+        self.accuracy = accuracy
+
+        # Variables used then caching the Ewald result
+        self.__last_structure = None
+        self.__last_ewald = None
+
+    def featurize(self, strc, site):
+
```

Comment:
Docstring the two params (strc and site)

Also, the other featurizers use ``idx`` as the site param (although we might want to rename everything to site_idx)